src/current/_includes/v25.2/known-limitations/rls-visibility-issue.md

```diff
@@ -0,0 +1,52 @@
+CockroachDB's row-level security (RLS) implementation exposes metadata about the number of restricted rows in a table under certain conditions, such as when users execute certain SQL functions. For example, when a user applies arbitrary SQL filters on a table with RLS enabled, it's possible for the user to see how many total rows are in the table; this count includes rows that the user does not have direct access to. This metadata leakage can also occur when statements like [`EXPLAIN ANALYZE`]({% link {{ page.version.version }}/explain.md %}) are used, as the output includes a count of the number of rows scanned by the query that can include the number of restricted rows, even though the rows themselves are not directly accessible. [#146952](https://github.com/cockroachdb/cockroach/issues/146952)
+
+For example, given the schema from {% if page.name == "row-level-security.md" %}[RLS for multi-tenant isolation](#rls-for-multi-tenant-isolation){% else %}[RLS for multi-tenant isolation]({% link {{ page.version.version }}/row-level-security.md %}#rls-for-multi-tenant-isolation){% endif %}, execute the following statements to view this behavior.
```

Comment:
The issue 146952 has good examples, so I don't know that we need to provide a full example here, unless that's standard practice for known limitations. Up to you though!